apps/android/LiveVideoApp/app/src/main/java/com/twilio/livevideo/app/base/LiveVideoApplication.kt

```diff
@@ -15,6 +16,7 @@ class LiveVideoApplication : Application() {
         super.onCreate()
 
         if (BuildConfig.DEBUG) {
+            SyncClient.setLogLevel(SyncClient.LogLevel.VERBOSE)
```

Comment:
Yeah, that's a good idea of having it with a lower log level, but in case of the SyncSDK, the default level is `SILENT` which means `Show no log messages.`. So, according to the log levels in Android, I changed to `DEBUG` to reduce the logs. 

FYI, These are the log levels we have in android SDKs:
```
        /** Show low-level tracing messages as well as all Debug log messages. */
        public static final int VERBOSE = android.util.Log.VERBOSE;

        /** Show low-level debugging messages as well as all Info log messages. */
        public static final int DEBUG = android.util.Log.DEBUG;

        /** Show informational messages as well as all Warning log messages. */
        public static final int INFO = android.util.Log.INFO;

        /** Show warnings as well as all Critical log messages. */
        public static final int WARN = android.util.Log.WARN;

        /** Show critical log messages as well as all Fatal log messages. */
        public static final int ERROR = android.util.Log.ERROR;

        /** Show fatal errors only. */
        public static final int ASSERT = android.util.Log.ASSERT;

        /** Show no log messages. */
        public static final int SILENT = Logger.SILENT;
```